cves/kernel/CVE-2012-2375.yml

```diff
@@ -167,8 +188,11 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: >-
+    It would require the remote server to send too much data so that the system
+    to crash. A developer would need to notice this happening or a test case was
+    needed for this.
```

Comment:
Can't a fuzzer just send a really big chunk of info?

Reply:
Updated